frontend/src/features/public-form/mutations.ts

```diff
@@ -98,25 +98,16 @@ export const usePublicFormMutations = (
     },
   )
 
-  const submitFormFeedbackMutation = useMutation(
-    (args: SubmitFormFeedbackBodyDto) =>
-      submitFormFeedback(formId, submissionId, args),
-    {
-      onError: (error: Error) => {
-        toast({ status: 'danger', description: error.message })
-      },
-    },
-  )
-
-  const submitStorageModeClearFormWithVirusScanningMutation = useMutation(
-    async (args: Omit<SubmitStorageFormClearArgs, 'formId'>) => {
+  const useSubmitClearFormWithVirusScanningMutation = (f: any) =>
```

Comment:
Note: very tricky typing here

Reply:
me: "mom can we have proper types please??"
mom: "we have types at home"

types at home: `any`